Aulas de Revisão/Revisão 2 entrega 19-09/planner/src/pages/PlannerPage.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { PlannerContainer } from './styled'
+import HeaderContainer from '../components/Header/Header'
+import GridWeek from '../components/GridWeek/GridWeek'
+import axios from "axios";
+import { BASE_URL } from "../constants/urls";
+
+const PlannerPage = (props, initialState) => {
```

Comment:
Não devemos colocar um segundo parametro no componente funcional, apenas o primeiro que é o objeto de props. Isso porque independente de quantas props passamos na chamada do componente, todos serão colocados dentro do objeto de props. Logo, se quiser uma props de initialState deve declara aqui somente a props, e quando for usar esse valor chamar como `props.initialState`